lib/simplecov/defaults.rb

```diff
@@ -30,28 +30,7 @@
   SimpleCov.at_exit.call
```

Comment:
There is `@exit_status = SimpleCov.exit_status_from_exception` above here - I don't think exit_status needs to be an instance variable at this point. maybe ever, Not sure